libexec/Makefile

```diff
@@ -84,9 +84,6 @@ _tftp-proxy=	tftp-proxy
 
 .if !defined(NO_PIC) && !defined(NO_RTLD)
 _rtld-elf=	rtld-elf rtld-elf-debug
-.if ${MACHINE_ABI:Mpurecap} && ${MACHINE_CPUARCH} == "aarch64"
```

Comment:
For this commit, you also want to add an entry to the top-level ObsoleteFiles.inc to remove the old files